internal/config/config.go

```diff
@@ -7,8 +7,27 @@ import (
 	"github.com/relab/hotstuff/internal/proto/orchestrationpb"
 )
 
-// Config holds the configuration for an experiment.
-type Config struct {
+// ReplicaMap is a map from host to a list of replica options.
+type ReplicaMap map[string][]*orchestrationpb.ReplicaOpts
+
+// ReplicaIDs converts the an entry map from []hotstuff.ID to []uint32.
+func (r ReplicaMap) ReplicaIDs(host string) hotstuff.IDSlice {
+	ids := make([]hotstuff.ID, 0, len(r[host]))
+	for _, opts := range r[host] {
+		ids = append(ids, hotstuff.ID(opts.ID))
+	}
+	return ids
+}
+
+type ClientMap map[string][]hotstuff.ID
+
+// ClientIDs converts the an entry map from []hotstuff.ID to []uint32.
+func (c ClientMap) ClientIDs(host string) hotstuff.IDSlice {
```

Comment:
This method should probably return a `[]uint32` directly, eliminating the need to call the conversion function later.